samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/SubmitToGradingActionListener.java

```diff
@@ -728,7 +728,7 @@ private void prepareItemGradingPerItem(ActionEvent ae, DeliveryBean delivery,
 						adds.add(itemgrading);
 				    }	
 				}
-				else if (StringUtils.isNotBlank(s)) {
```

Comment:
Original logic from 2006:
`} else if (itemgrading.getAnswerText() != null && !itemgrading.getAnswerText().equals("")) {`
which matches StringUtils.isNotBlank or more closely StringUtils.isNotEmpty

Would be nice if you could provide some explanation of why SAK-43598 created this regression

Reply:
There is more info on the jira: https://jira.sakaiproject.org/browse/SAK-44530